--- cryptopy-1.2.5/binascii_plus.py
+++ cryptopy-1.2.5/binascii_plus.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ binascii_plus
 
     Enhanced hex packet input and display
--- cryptopy-1.2.5/crypto/app/filecrypt.py
+++ cryptopy-1.2.5/crypto/app/filecrypt.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ cipher.app.filecrypt
 
--- cryptopy-1.2.5/crypto/app/__init__.py
+++ cryptopy-1.2.5/crypto/app/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.app
 
     Sample applications of CryptoPy
--- cryptopy-1.2.5/crypto/cipher/aes_cbc.py
+++ cryptopy-1.2.5/crypto/cipher/aes_cbc.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.aes_cbc
 
     AES_CBC Encryption Algorithm
--- cryptopy-1.2.5/crypto/cipher/aes_cbc_test.py
+++ cryptopy-1.2.5/crypto/cipher/aes_cbc_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.cbc_test
 
--- cryptopy-1.2.5/crypto/cipher/aes.py
+++ cryptopy-1.2.5/crypto/cipher/aes.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.aes
 
     AES Encryption Algorithm
--- cryptopy-1.2.5/crypto/cipher/aes_sbox_analysis.py
+++ cryptopy-1.2.5/crypto/cipher/aes_sbox_analysis.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.aes_sbox_analysis
 
--- cryptopy-1.2.5/crypto/cipher/aes_test.py
+++ cryptopy-1.2.5/crypto/cipher/aes_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.aes_test
 
--- cryptopy-1.2.5/crypto/cipher/arc4.py
+++ cryptopy-1.2.5/crypto/cipher/arc4.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.arc4
 
     A Stream Cipher Encryption Algorithm 'Arcfour'
--- cryptopy-1.2.5/crypto/cipher/arc4_test.py
+++ cryptopy-1.2.5/crypto/cipher/arc4_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.arc4_test
 
--- cryptopy-1.2.5/crypto/cipher/base.py
+++ cryptopy-1.2.5/crypto/cipher/base.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.base
 
 
--- cryptopy-1.2.5/crypto/cipher/cbc.py
+++ cryptopy-1.2.5/crypto/cipher/cbc.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.cbc
 
     CBC mode of encryption for block ciphers.
--- cryptopy-1.2.5/crypto/cipher/cbc_test.py
+++ cryptopy-1.2.5/crypto/cipher/cbc_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.cbc_test
 
--- cryptopy-1.2.5/crypto/cipher/ccm.py
+++ cryptopy-1.2.5/crypto/cipher/ccm.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.ccm
 
     CCM block cipher mode
--- cryptopy-1.2.5/crypto/cipher/ccm_test.py
+++ cryptopy-1.2.5/crypto/cipher/ccm_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.ccm_test
 
--- cryptopy-1.2.5/crypto/cipher/icedoll.py
+++ cryptopy-1.2.5/crypto/cipher/icedoll.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.icedoll
 
     Modification of Rijndael to provide infinite error extension.
--- cryptopy-1.2.5/crypto/cipher/icedoll_test.py
+++ cryptopy-1.2.5/crypto/cipher/icedoll_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.icedoll_test
 
--- cryptopy-1.2.5/crypto/cipher/__init__.py
+++ cryptopy-1.2.5/crypto/cipher/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher
 
     cipher package of CryptoPy
--- cryptopy-1.2.5/crypto/cipher/rijndael.py
+++ cryptopy-1.2.5/crypto/cipher/rijndael.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.rijndael
 
     Rijndael encryption algorithm
--- cryptopy-1.2.5/crypto/cipher/rijndael_test.py
+++ cryptopy-1.2.5/crypto/cipher/rijndael_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.rijndael_test
 
--- cryptopy-1.2.5/crypto/cipher/test_all_ciphers.py
+++ cryptopy-1.2.5/crypto/cipher/test_all_ciphers.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.test_all_ciphers
 
--- cryptopy-1.2.5/crypto/cipher/tkip_encr.py
+++ cryptopy-1.2.5/crypto/cipher/tkip_encr.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.tkip_encr
 
     TKIP encryption from IEEE 802.11 TGi
--- cryptopy-1.2.5/crypto/cipher/tkip_encr_test.py
+++ cryptopy-1.2.5/crypto/cipher/tkip_encr_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.tkip_encr_test
 
--- cryptopy-1.2.5/crypto/cipher/tkip_fake_crc_test.py
+++ cryptopy-1.2.5/crypto/cipher/tkip_fake_crc_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.tkip_fake_crc_test
 
--- cryptopy-1.2.5/crypto/cipher/trolldoll.py
+++ cryptopy-1.2.5/crypto/cipher/trolldoll.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.trolldoll
 
     Modification to Icedoll to take advantage of the better error extension
--- cryptopy-1.2.5/crypto/cipher/wep.py
+++ cryptopy-1.2.5/crypto/cipher/wep.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.cipher.wep
 
     The WEP encryption algorithm used in IEEE 802.11
--- cryptopy-1.2.5/crypto/cipher/wep_test.py
+++ cryptopy-1.2.5/crypto/cipher/wep_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.wep_test
 
--- cryptopy-1.2.5/crypto/common.py
+++ cryptopy-1.2.5/crypto/common.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.common
     Common utility routines for crypto modules
 
--- cryptopy-1.2.5/crypto/entropy/__init__.py
+++ cryptopy-1.2.5/crypto/entropy/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.entropy
 
     Entropy package of CryptoPy
--- cryptopy-1.2.5/crypto/entropy/pagingEntropy.py
+++ cryptopy-1.2.5/crypto/entropy/pagingEntropy.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.entropy.pagingEntropy
 
     Uses variations in disk access time to generator entropy.  A long string is
--- cryptopy-1.2.5/crypto/entropy/prn_rijndael.py
+++ cryptopy-1.2.5/crypto/entropy/prn_rijndael.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.entropy.prn_rijndael
 
     A Psudeo Random Number Generator based on Rijndael_256k_256b
--- cryptopy-1.2.5/crypto/entropy/prn_rijndael_test.py
+++ cryptopy-1.2.5/crypto/entropy/prn_rijndael_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.entropy.prn_rijndael_test
 
--- cryptopy-1.2.5/crypto/errors.py
+++ cryptopy-1.2.5/crypto/errors.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ errors
     Error classes for cryptographic modules
 
--- cryptopy-1.2.5/crypto/hash/hash.py
+++ cryptopy-1.2.5/crypto/hash/hash.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """   crypto.cipher.hash
       Base class for cryptographic hash algorithms
       An alternate interface (no 'new').
--- cryptopy-1.2.5/crypto/hash/__init__.py
+++ cryptopy-1.2.5/crypto/hash/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ The crypto.hash package.
     Part of the CryptoPy framework.
 """   
--- cryptopy-1.2.5/crypto/hash/md5Hash.py
+++ cryptopy-1.2.5/crypto/hash/md5Hash.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """   md5Hash.py
       Wrapper for python md5 module to support crypo module standard interface
 """
--- cryptopy-1.2.5/crypto/hash/md5Hash_test.py
+++ cryptopy-1.2.5/crypto/hash/md5Hash_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ md5Hash_test.py
     Unit tests for md5Hash.py  (not the default python library!)
--- cryptopy-1.2.5/crypto/hash/sha1Hash.py
+++ cryptopy-1.2.5/crypto/hash/sha1Hash.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.hash.sha1Hash
 
     Wrapper for python sha module to support crypo module standard interface
--- cryptopy-1.2.5/crypto/hash/sha1Hash_test.py
+++ cryptopy-1.2.5/crypto/hash/sha1Hash_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ sha1Hash_test.py
     Unit tests for sha1.py
--- cryptopy-1.2.5/crypto/icedoll_test.py
+++ cryptopy-1.2.5/crypto/icedoll_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.cipher.icedoll_test
 
--- cryptopy-1.2.5/crypto/__init__.py
+++ cryptopy-1.2.5/crypto/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto
     CryptoPy - pure python cryptographic libraries
 
--- cryptopy-1.2.5/crypto/keyedHash/hmacHash.py
+++ cryptopy-1.2.5/crypto/keyedHash/hmacHash.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """  hmacHash.py
 
     Implemention of Request for Comments: 2104
--- cryptopy-1.2.5/crypto/keyedHash/hmacHash_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/hmacHash_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ hmacHash_test.py
     Unit tests for hmacHash.py
--- cryptopy-1.2.5/crypto/keyedHash/__init__.py
+++ cryptopy-1.2.5/crypto/keyedHash/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.keyedHash
 
     The keyedHash package.
--- cryptopy-1.2.5/crypto/keyedHash/michael.py
+++ cryptopy-1.2.5/crypto/keyedHash/michael.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.keyedHash.michael
 
     A reference implementation of the Michael Message Integrety Chek (MIC)
--- cryptopy-1.2.5/crypto/keyedHash/michael_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/michael_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ crypto.keyedHash.michael_test
 
--- cryptopy-1.2.5/crypto/keyedHash/pbkdf2.py
+++ cryptopy-1.2.5/crypto/keyedHash/pbkdf2.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.keyedHash.pbkdf2
 
 
--- cryptopy-1.2.5/crypto/keyedHash/pbkdf2_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/pbkdf2_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ crypto.keyedHash.pbkdf2_test
 
--- cryptopy-1.2.5/crypto/keyedHash/prf_dot11.py
+++ cryptopy-1.2.5/crypto/keyedHash/prf_dot11.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """  prf_dot11.py
 
 The PRF function is used in a number of places in 802.11i
--- cryptopy-1.2.5/crypto/keyedHash/prf_dot11_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/prf_dot11_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """  crypto.keyedHash.prf_dot11_test
 	 Tests of the IEEE 802.11 PRF functions
--- cryptopy-1.2.5/crypto/keyedHash/tkip_key_mixing.py
+++ cryptopy-1.2.5/crypto/keyedHash/tkip_key_mixing.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.keyedHash.tkip_key_mixing.py
     TKIP Temporal Key Mixing Function reference implementation
 
--- cryptopy-1.2.5/crypto/keyedHash/tkip_key_mixing_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/tkip_key_mixing_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.keyedHash.tkip_key_mixing
     Tests for TKIP key mixing function
--- cryptopy-1.2.5/crypto/keyedHash/tkip_mic.py
+++ cryptopy-1.2.5/crypto/keyedHash/tkip_mic.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.keyedHash.tkip_mic
 
     A reference implementation of the TKIP Message Integrety Chek (MIC)
--- cryptopy-1.2.5/crypto/keyedHash/tkip_mic_test.py
+++ cryptopy-1.2.5/crypto/keyedHash/tkip_mic_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ Testing of TKIP_MIC Class
 """
--- cryptopy-1.2.5/crypto/passwords/__init__.py
+++ cryptopy-1.2.5/crypto/passwords/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ CryptoPy - a pure python cryptographic libraries
 
     crypto.passwords package
--- cryptopy-1.2.5/crypto/passwords/passwordfactory.py
+++ cryptopy-1.2.5/crypto/passwords/passwordfactory.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ crypto.passwords.passwordfactory
 
     Python classes to create and recover passwords.  Currently contains
--- cryptopy-1.2.5/crypto/passwords/passwordfactory_test.py
+++ cryptopy-1.2.5/crypto/passwords/passwordfactory_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ crypto.passwords.passwordfactory_test
 
--- cryptopy-1.2.5/fmath/__init__.py
+++ cryptopy-1.2.5/fmath/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ fmath module
 
     Copyright © (c) 2002 by Paul A. Lambert
--- cryptopy-1.2.5/fmath/prime.py
+++ cryptopy-1.2.5/fmath/prime.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ fmath.prime
 
     Start of prime number routines. Rabin-miller test works, more to come later.
--- cryptopy-1.2.5/fmath/primes_to_5k.py
+++ cryptopy-1.2.5/fmath/primes_to_5k.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ fmath.primes_to_5k
 
     All of the prime numbers below 5000
--- cryptopy-1.2.5/fmath/prime_test.py
+++ cryptopy-1.2.5/fmath/prime_test.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #! /usr/bin/env python
 """ fmath.prime_test
 
--- cryptopy-1.2.5/__init__.py
+++ cryptopy-1.2.5/__init__.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 """ cryptopy
     CryptoPy - pure python cryptographic libraries
 
--- cryptopy-1.2.5/setup.py
+++ cryptopy-1.2.5/setup.py
@@ -1,3 +1,4 @@
+# coding: utf-8
 #!/usr/bin/env python
 """ cryptopy setup
 
